### PR TITLE
Add libraries page in analytics section

### PR DESCRIPTION
**Pull request by harrisred**
* Add wordpress/woocommerce plugin
* Add libraries to related resources in other analytics pages

### CODE REVIEW

File: source/localizable/analytics/libraries.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+> ##### Note
+> The product ID must be the same ID used in the XML feed provided to <%= flavor.capitalize %>.
+
+## Related Resources
+
+* Configuration
+  * <%= link_to 'Analytics Settings', 'settings' %>
+  * <%= link_to 'Ecommerce metrics', 'ecommerce' %>
+* <%= link_to 'Examples', 'examples' %>
```
Comment (Zorbash) [inline source/localizable/analytics/libraries.html.md.erb]: indentation

File: source/localizable/analytics/libraries.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+## Wordpress / WooCommerce
+
+You can find the plugin <%= link_to 'here', 'https://wordpress.org/plugins/skroutz-analytics-woocommerce/', target: '_blank' %>.
+
+Requires to have Wordpress 4.0 or higher and the <%= link_to 'WooCommerce plugin', 'https://wordpress.org/plugins/woocommerce/', target: '_blank' %> installed.
+
+### Automatic installation (recommended)
+
+1. Navigate to your wordpress: `Admin panel > Plugins > Add New`
+2. Search this plugin by its name: *Skroutz Analytics WooCommerce*
```
Comment (Zorbash) [inline source/localizable/analytics/libraries.html.md.erb]: s/by its name/by name

File: source/localizable/analytics/libraries.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+3. Install the plugin.
```
Comment (Zorbash) [inline source/localizable/analytics/libraries.html.md.erb]: Remove `.`.

File: source/localizable/analytics/libraries.html.md.erb [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+4. Activate the plugin.
+
+### Manual installation
```
Comment (Zorbash) [inline source/localizable/analytics/libraries.html.md.erb]: Why would anyone not proceed with the automatic installation?

File: source/localizable/analytics/libraries.html.md.erb [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+1. Download the <%= link_to 'plugin file', 'https://wordpress.org/plugins/skroutz-analytics-woocommerce/', target: '_blank' %> to your computer and unzip it.
+2. Using an FTP client, or your hosting control panel, upload the unzipped plugin folder to your WordPress installation’s directory: `wp-content/plugins/`
```
Comment (Zorbash) [inline source/localizable/analytics/libraries.html.md.erb]: What about uploading the zip and then unzipping to the desired dir?

File: source/localizable/analytics/libraries.html.md.erb [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+3. Activate the plugin through the Plugins menu in WordPress admin.
+
+### Configuration
```
Comment (Zorbash) [inline source/localizable/analytics/libraries.html.md.erb]: Maybe we can avoid having duplicate documentation about config options and point them to the wordpress plugins page doc.

**Comment by Zorbash** [comment]
+1 LGTM

File: source/localizable/analytics/libraries.html.md.erb (hunk 1 above)
Comment (harrisred) [inline source/localizable/analytics/libraries.html.md.erb]: The indentation is on purpose like this. There are two sections: Configuration (with two subsections) and Examples. This is how you have it the other pages as well.

Comment (Zorbash) [inline source/localizable/analytics/libraries.html.md.erb]: :ok_hand:

**Comment by Zorbash** [comment]
Some visual diff of the PR.

selection_225


selection_226

**Pull request merged**